.golangci.yml

```diff
@@ -46,7 +46,6 @@ output:
 
 linters:
   enable:
```

Comment:
Something weird has happened with your git history -- there are all this extra changes in here.

You might want to rebase your local branch against master, and squash your changes, and force push over the top to make sure you're in a happy place.

Reply:
Ok, everything looks fine now. Let me know if there's anything else I need to do (as you might have guessed, I'm not a git wizard!)